charts/Controls.tsx

```diff
@@ -1058,6 +1074,20 @@ export class ControlsFooterView extends React.Component<{
             <SettingsMenu chart={chart} onDismiss={this.onSettingsMenu} />
         )
 
+        const relatedQuestionElement = relatedQuestions && hasRelatedQuestion && (
+            <div className="relatedQuestion">
+                Related:&nbsp;
+                <a
+                    href={relatedQuestions[0].url}
+                    target="_blank"
```

Comment:
```suggestion
                    target="_blank"
                    rel="noopener"
```

Ah, one last thing that came to mind here: Let's make this [`rel="noopener"`](https://mathiasbynens.github.io/rel-noopener/) since it can point to non-ourworldindata URLs potentially.